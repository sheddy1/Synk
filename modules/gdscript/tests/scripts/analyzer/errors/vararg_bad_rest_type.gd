func f(...args: int):
	pass

func test():
	pass
