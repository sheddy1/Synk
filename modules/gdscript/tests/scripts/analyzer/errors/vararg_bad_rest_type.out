GDTEST_ANALYZER_ERROR
The rest parameter type must be "Array", but "int" is specified.
